docs/src/components/SearchInput/primitives/StyledWrapper.ts

```diff
@@ -1,10 +1,13 @@
 import styled, { css } from "styled-components";
 
-import { Size } from "..";
+export enum Size {
+  Regular,
+  Large,
+}
 
 const StyledWrapper = styled.div<{ size: Size }>`
   position: relative;
-  min-width: 24em; /* so that the placeholder fits in */
+  max-width: 24em; /* so that the placeholder fits in */
```

Comment:
i think for that kind of values we have to use pixels

Reply:
I used ems because I based it on placeholder, which is character count, so it makes sense to me. If there's a wider typeface, I'd like this value to be wider.